source/configure_project/frontmatter/index.html.md.erb

```diff
@@ -66,6 +66,16 @@ hide_in_navigation: true
 ---
 ```
 
+### prevent_indexing
```

Comment:
The h3s here are in alphabetical order, so this would belong under the '\#\#\# parent' section